test/e2e/autoscale_test.go

```diff
@@ -90,9 +91,10 @@ func generateTraffic(clients *test.Clients, logger *logging.BaseLogger, concurre
 					}
 
 					if res.StatusCode != http.StatusOK {
+						logger.Errorf("request %d failed", requestID)
 						logger.Errorf("non 200 response %v", res.StatusCode)
 						logger.Errorf("response headers: %v", res.Header)
-						logger.Errorf("response body: %v", res.Body)
+						logger.Errorf("response body: %v", string(res.Body[:]))
```

Comment:
That's the first time I've seen `string(res.Body[:])` as a way to get a string from the response body. Something like `string(ioutil.ReadAll(res.Body))` is how I've usually seen it done. Since I know you were in pkg/spoof recently, here's an example from there - https://github.com/knative/pkg/blob/1a50a39daf360e154f948b654caa1987242468c1/test/spoof/spoof.go#L142

You'll also have to import `"io/ioutil"` to use this method.

Reply:
Good point, I dug around a bit:

1. I can just drop the `[:]` magic.
2. In this case `ioutil.ReadAll` is overkill because I already have a value of type `[]byte` (which ReadAll produces). We're using a spoofed client here, which alters the signature of the Response a little bit.

Pushed an update, thanks for the suggestion!